tests/integration/helpers/image-type-test.js

```diff
@@ -0,0 +1,17 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Helper | image-type', function(hooks) {
+  setupRenderingTest(hooks);
+
+  // Replace this with your real tests.
```

Comment:
did you mean to add this test?

Reply:
ya, it's just a generated one. While it's a prototype I'm not bothering with tests, but we def need to unit test the service, soon